packages/common/test/compose.test.ts

```diff
@@ -196,83 +196,123 @@ describe("composeContent", () => {
         );
       });
     });
-  });
```

Comment:
The changes on this file look like a lot, but if you use the vs code view (press `.`), you'll see that I moved a parenthesis/bracket pair. Turns out that `publishedDate` tests were just outside of the describe that defined `item`, so undefined was being passed in.